### ISSUE TITLE
ESPHome 2024.6.3 - 'ota' requires a 'platform' key but it was not specified.

### ISSUE THREAD

**Issue by Belaial** [hn/ginlong-solis#39]
Small issue, easy fix, would do a pull request but not sure how to do those since I am a github-noob 😄 

In ESPHome 2024.6.3 (maybe earlier also) there is a new line needed in

https://github.com/hn/ginlong-solis/blob/master/solis-esphome-emw3080.yaml
https://github.com/hn/ginlong-solis/blob/master/solis-esphome-esp8266.yaml

This
```
ota:
  password: !secret ota_password
```

Needs to be this

```
ota:
- platform: esphome
  password: !secret ota_password
```

So just add "- platform: esphome" and that should be all as far as I know 👍

**Comment by Belaial** [comment]
I tried now to make a PL ( https://github.com/hn/ginlong-solis/pull/40 ), hopefully it's done like it should be, my first ever so..... 😃 sorry if it's wrong.

**Comment by hn** [comment]
Thanks @Belaial and welcome to the wonderful world of pull requests ;-)

Merged #40

**Comment by Belaial** [comment]
Thank you! Happy to help! 😄
